test/tests.js

```diff
@@ -593,11 +640,94 @@ describe( 'passport-saml /', function() {
       };
     }
 
+    function testForCheckLogout( check ) {
+      return function( done ) {
+        var app = express();
+        app.use( bodyParser.urlencoded({extended: false}) );
+        app.use( passport.initialize() );
+        var config = check.config;
+        config.callbackUrl = 'http://localhost:3033/login';
+        config.entryPoint = 'https://wwwexampleIdp.com/saml';
```

Comment:
Purposely no dot between 'www' and 'exampleIdp'?

Reply:
I'd say a typo, but it comes from the upstream.